source/how-tos/create-ias-boot-image.rst

```diff
@@ -0,0 +1,116 @@
+.. _create-ias-boot-image:
+
+Create IAS Boot Image
+----------------------
+
+This page provides intructions to create a bootable Linux kernel image in IAS format and setup boot partition in |SPN| to boot from it.
+
+
+The following step-by-step instructions is an example to boot Yocto kernel using IAS image format on |UP2|.
+
+
+**Step 1:** Download and install open source IAS tool from |IAS|
+
+.. |IAS| raw:: html
+
+   <a href="https://github.com/intel/iasimage" target="_blank">here</a>
+
+
+**Step 2:** Get kernel and associated files from target Yocto disk image.
+
+Copy vmlinuz (e.g. ``vmlinuz``), initramfs (``initrd``) and get kernel command line string (e.g. /proc/cmdline) and transfer them to your development host.

```

Comment:
we should state how to get kernel command line string.   You can provide an example for yocto.  It is in grub.conf.